test/CrowdsaleGenTest.js

```diff
@@ -62,15 +62,18 @@ contract('LifCrowdsale Property-based test', function(accounts) {
     help.debug("crowdsaleTestInput data:\n", input, publicPresaleStartTimestamp, publicPresaleEndTimestamp, startTimestamp, end1Timestamp, end2Timestamp);
 
     let {publicPresaleRate, rate1, rate2, owner, setWeiLockSeconds} = input.crowdsale,
-      ownerAddress = accounts[input.crowdsale.owner];
+      ownerAddress = gen.getAccount(input.crowdsale.owner),
+      foundationWallet = gen.getAccount(input.crowdsale.foundationWallet);
     let shouldThrow = (publicPresaleRate == 0) ||
       (rate1 == 0) ||
       (rate2 == 0) ||
       (publicPresaleStartTimestamp >= publicPresaleEndTimestamp) ||
       (publicPresaleEndTimestamp >= startTimestamp) ||
       (startTimestamp >= end1Timestamp) ||
       (end1Timestamp >= end2Timestamp) ||
-      (setWeiLockSeconds == 0)
+      (setWeiLockSeconds == 0) ||
+      (ownerAddress == 0) ||
```

Comment:
Here we are comparing against the address? not the address index right?

In case its against address it should be `0x0000000000000000000000000000000000000000` instad of `0`